docs/TypeScript.md

```diff
@@ -0,0 +1,116 @@
+<h1 align="center">Fastify</h1>
+
+## TypeScript
+Fastify is shipped with the a typings file so it should "just work" when your used in a TypeScript application.
+
+If you're using plugins be sure to read the sections below.
```

Comment:
How about this instead?: `Plugins may or may not include typings. See [Plugin Types](#plugin-types) for more information.`